src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -668,14 +668,22 @@ private void LoadAssemblies(IEnumerable<string> assemblies)
         /// </summary>
         private static IEnumerable<PortableExecutableReference> InitDefaultRefAssemblies()
         {
-            // netcoreapp3.0 currently comes with 148 reference assemblies (maybe more in future), so we use a capacity of '150'.
+            // PowerShell is distributed with 149 reference assemblies, so we use a capacity of '150'.
             var defaultRefAssemblies = new List<PortableExecutableReference>(150);
```

Comment:
We could define a const because we use the value below in Assert.
We could remove the value from the comment.
The comment could mention .Net version so that we understand where the baseline comes from.